tests/testthat/test_quality_check.R

```diff
@@ -71,63 +71,67 @@ test_that("Output reports are not created...", {
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B6b")$Error[c(1, 152, 153, 154)], c(NA, NA, TRUE, TRUE))
 
-  # Test 7: B6c
+  # Test check B6c
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B6c")$Warning[c(1, 152, 153, 154)], c(NA, TRUE, NA, NA))
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B6c")$Error[c(1, 152, 153, 154)], c(NA, NA, TRUE, TRUE))
 
-  # Test 8: B6d
+  # Test check B6d
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B6d")$Warning[c(1, 152, 153, 154, 155)], c(NA, TRUE, TRUE, NA, NA))
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B6d")$Error[c(1, 152, 153, 154, 155)], c(NA, NA, NA, TRUE, TRUE))
 
-  # Test 9: B7
+  # Test check B7
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B7")$Warning, c(NA, TRUE, NA))
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B7")$Error, c(NA, NA, TRUE))
 
-  # Test 10: B8
+  # Test check B8
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B8")$Warning, c(NA, NA, NA))
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B8")$Error, c(NA, TRUE, TRUE))
 
-  # Test 11: B9
+  # Test check B9
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B9")$Warning, c(NA, NA, NA, NA, NA, NA))
   expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B9")$Error, c(NA, NA, NA, NA, NA, TRUE))
 
-  # Test 12: B10
-  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B10")$Warning, c(NA, TRUE))
-  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B10")$Error, c(NA, NA))
+  # Test check B10
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B10")$Warning, c(NA, TRUE, NA))
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B10")$Error, c(NA, NA, TRUE))
 
-  # Test 13: B11
-  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B11")$Warning, c(NA, TRUE))
-  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B11")$Error, c(NA, NA))
+  # Test check B11
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B11")$Warning, c(NA, TRUE, NA))
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B11")$Error, c(NA, NA, TRUE))
 
-  # Test 14: B12
-  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B12")$Warning, c(NA, TRUE))
-  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B12")$Error, c(NA, NA))
+  # Test check B12
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B12")$Warning, c(NA, TRUE, NA))
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B12")$Error, c(NA, NA, TRUE))
 
-  # Test 15: C2a
+  # Test check B13
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B13")$Warning, c(NA, NA))
+  expect_equal(subset(dummy_check$R_data$Brood_data, CheckID == "B13")$Error, c(NA, TRUE))
```

Comment:
These tests are all failing for me. Are they working on your side?

Reply:
They are working here, strangely enough. I suspect it might be related to the dummy_data not being created/updated properly.
```
== Results =================================================================================================
Duration: 1280.4 s

-- Skipped tests  ------------------------------------------------------------------------------------------
* Skipped in devtools::test(). Run manually to test archiving (2)

[ FAIL 0 | WARN 0 | SKIP 2 | PASS 1531 ]
```

Reply:
Ok, I'll try running the dummy data code on my side and try again.